Makefile

```diff
@@ -186,14 +186,18 @@ stylesheets: public/css/index.css
 
 .IGNORE: public/css/index.css
 public/css/index.css: $(STYLESHEETS)
-	lessc $< $@
+	@hash lessc > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
+		docker run --rm -v $(PWD):/app appleboy/node-less $< $@; \
+	else \
+		lessc $< $@; \
+	fi
```

Comment:
Not all golang developer have node environment so docker is better for everyone.